os/fs/procfs/fs_procfsproc.c

```diff
@@ -80,12 +81,15 @@
 #include <tinyara/fs/fs.h>
 #include <tinyara/fs/procfs.h>
 #include <tinyara/fs/dirent.h>
+#include <tinyara/pm/pm.h>
 
 #ifdef CONFIG_SCHED_CPULOAD
 #include <tinyara/clock.h>
 #endif
 
 #include <arch/irq.h>
+#include <pm.h>
```

Comment:
How about this change?
`
#if defined(CONFIG_PM) && defined(CONFIG_PM_METRICS)
	#include <pm.h>
#endif
`
and line 84

Reply:
I agree.
I will update soon.